app/views/wishlists/show.html.erb

```diff
@@ -1,14 +1,27 @@
-<h1><%= @wishlist.name %></h1>
+<div class="container">
+  <h1 class="wishlist-name"><%= @wishlist.name %></h1>
 
-<p><%= @wishlist.description %></p>
+  <p class="wishlist-description"><%= @wishlist.description %></p>
 
-<p>
-  <strong>Your unique link for this whislist:</strong><br>
-  <%= wishlists_for_guest_url(@wishlist.token) %>
-</p>
+  <div class="input-group wishlist-url-display input-group-sm">
+    <span class="input-group-addon">Link for sharing:</span>
+    <input type="text" class="form-control" id="unique_url" value="<%= wishlists_for_guest_url(@wishlist.token) %>" readonly>
+    <span class="input-group-addon btn" id="clip" data-clipboard-target="#unique_url">copy</span>
+  </div>
+  <div class="wishlist-main-buttons">
```

Comment:
I had empty lines between `<div>`s in previous version and was advised to change it. ;-)